README.md

```diff
@@ -6,7 +6,114 @@
 [![Coverage Status](https://coveralls.io/repos/github/server-state/systemd-module/badge.svg?branch=master)](https://coveralls.io/github/server-state/systemd-module?branch=master)
 ![module type: official](https://img.shields.io/badge/module%20type-official-%23015ba0)
 
-A module to view the current status of your systemd services.
+### Description
+
+A module to view the current status of your systemd services. Its response is a object containing keys named as given unit names in [SMF](https://github.com/server-state/specs/blob/master/terminology/server-module-function.md) options with another object as value.
+This object contains the properties of the given unit as key/value pairs, for example:
+```json
```

Comment:
That's `js`. For `json`, you need double quotes around all keys etc. :wink:

Reply:
Fixed.